src/backend/src/modules/pia-intake/metadata/pia-status.metadata.ts

```diff
@@ -283,8 +283,63 @@ export const piaStatusMetadata: Partial<
           },
         ],
       },
+      [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+        allow: true,
```

Comment:
<optional> Do you think if we should add field validations to check if all Program Area reviews are submitted? And only then you're allowed to move to PENDING_COMPLETION status